lib/litani_report.py

```diff
@@ -319,22 +317,30 @@ def render(run, report_dir):
     with litani.atomic_write(report_dir / "index.html") as handle:
         print(page, file=handle)
 
-    with litani.atomic_write(report_dir / litani.RUN_FILE) as handle:
+    with litani.atomic_write(temporary_report_dir / litani.RUN_FILE) as handle:
         print(json.dumps(run, indent=2), file=handle)
 
     pipe_templ = env.get_template("pipeline.jinja.html")
     for pipe in run["pipelines"]:
         page = pipe_templ.render(run=run, pipe=pipe)
-        with litani.atomic_write(report_dir / pipe["url"]) as handle:
+        with litani.atomic_write(temporary_report_dir / pipe["url"]) as handle:
             print(page, file=handle)
 
+    temp_symlink_dir = report_dir.with_name(report_dir.name + "-tmp")
```

Comment:
For total safety, I recommend to not use a fixed `-tmp` suffix. If this method somehow gets called twice at the same time, one of the `rename` calls will fail. Instead, use a UUID as a suffix. Since it's a temporary directory that you're going to rename in just a moment, it doesn't really matter. And if this gets called twice in parallel, one of the calls will 'win' without any failure.